apps/astarte_housekeeping_api/lib/astarte_housekeeping_api_web/controllers/realm_controller.ex

```diff
@@ -29,8 +29,12 @@ defmodule Astarte.Housekeeping.APIWeb.RealmController do
     render(conn, "index.json", realms: realms)
   end
 
-  def create(conn, %{"data" => realm_params}) do
-    with {:ok, %Realm{} = realm} <- Realms.create_realm(realm_params) do
+  def create(conn, %{"data" => realm_params} = info) do
+    # booleans are atoms
+    async_operation = Map.get(info, "async_operation", "true") |> String.to_existing_atom()
```

Comment:
Couple of observations:
- `info` -> `params` (that's their actual name)
- While legitimate, the call to `to_existing_atom` is a little brutal. Either just match on `"true"` and put `false` otherwise with an `if`, or return some kind of informative error (the average user doesn't have access to logs so seeing a 500 won't help them to understand they misspelled `treu`). I'd honestly go for the first one since we already do so in other places (see, e.g., the `details` query param in `AppEngine.APIWeb.DeviceStatusController`).